build.gradle

```diff
@@ -75,6 +75,7 @@ subprojects {
         implementation "ch.qos.logback:logback-access:${depVersions.logbackClassic}"
         implementation "org.slf4j:jul-to-slf4j:${depVersions.julToSlf4j}"
         implementation "org.apache.httpcomponents:httpclient:${depVersions.apacheHttp}"
+        implementation "com.google.guava:guava:30.1.1-jre"
```

Comment:
I thought we removed this? Why did you add it back?

Reply:
I am using `com.google.common.collect.ImmutableList` from this library to generate the request object. So I added it.
check [here](https://github.com/commercetools/commercetools-payone-integration/pull/392/files#diff-5908522bce9ab5c456c3d4f15a715996870392cc55574720e636b1f6c0ddcdfcR198)

Reply:
Can you replace it with normal JDK list and remove guava? I don't think it's a good idea to add a library only for this purpose.

Reply:
Not really only for List. I am using guava library for few other things. Example `Preconditions` , creation on immutable map and immutable list. 
Do you see any issue by adding this library?

Reply:
Yes, there were issues with this library and we could not properly update it. That's why we removed it recently. See here: https://github.com/commercetools/commercetools-payone-integration/pull/370. And we're not alone with these problems: https://www.reddit.com/r/java/comments/d3gi2g/do_you_still_use_guave_and_if_you_dont_anymore/. So avoid it if you can.

Reply:
https://github.com/commercetools/commercetools-payone-integration/pull/392/commits/afaab776a051a57299e262fd3f59683fea7f5a08